src/defaultConfig.json

```diff
@@ -1,8 +1,9 @@
 {
+  "aliaVersion": "0.3.0",
```

Comment:
probably better to not include `aliaVersion` in `defaultConfig.json`, but add it using `project.version` when loaded into `config.js`